### ISSUE TITLE
Email integration for alerts

### ISSUE THREAD

**Issue by Kludex** [pydantic/logfire#80]
### Description

We support slack, and webhooks (with slack format only), but we still don't support email integration.

Please let us know if you are interested in this, so we can prioritize it.

**Comment by CherrySuryp** [comment]
Hello there, email notifications would be handy though. Moreover you can add notifications to Telegram via Telegram Bot API, this channel is also popular nowadays

**Comment by slingshotvfx** [comment]
+1 definitely interested in email notifications here

**Comment by schatimo** [comment]
What is the current status of this feature as it is now more than 2 years ago when it was initially posted?

**Comment by alexmojaki** [comment]
You can try it out by opening https://logfire-us.pydantic.dev/-/feature-flag (or change us->eu) and enabling email_notification.

**Comment by pedro-teixeira** [comment]
Email and Telegram would be a game-changer.

**Comment by pedro-teixeira** [comment]
> You can try it out by opening https://logfire-us.pydantic.dev/-/feature-flag (or change us->eu) and enabling email_notification.

It looks like this is a public page.

**Comment by alexmojaki** [comment]
Yes, those feature flags are for your browser only, so it's safe for the page to be public. I think all the backend wiring is in place but I could be wrong. I don't know how ready the feature is for production use.

**Comment by pedro-teixeira** [comment]
@alexmojaki looks like this feature was removed? I couldn't find any release note explaining the change.

**Comment by Kludex** [comment]
> [@alexmojaki](https://github.com/alexmojaki) looks like this feature was removed? I couldn't find any release note explaining the change.

Yes, go to https://logfire-eu.pydantic.info/settings/early-access. You can enable the email alerts early access.

---

I'll close this issue since email integration is in early access.

**Comment by pedro-teixeira** [comment]
Right, but the email option isn't in the early access section anymore either, so I'm wondering whether the feature was actually dropped.
